packages/forgae-cli/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "forgae",
+  "version": "1.4.0",
+  "description": "This is the default package.json generated for your project",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha './test/**/*.js' --timeout 0 --exit"
+  },
+  "bin": {
+    "forgae": "./forgae-cli.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "await-spawn": "^2.1.1",
+    "bignumber.js": "^8.1.1",
+    "chai": "^4.1.2",
+    "chai-as-promised": "^7.1.1",
+    "chai-files": "^1.4.0",
+    "chalk": "^2.4.1",
```

Comment:
Please double check all dependancies, because I am pretty sure you've moved color-utils to forgae-utils where chalk is used, not here. Also I did not see chalk in the utils dependancies list.